packages/browser/src/Widget/Server/Workspace/View/D3WorkspaceCanvas.js

```diff
@@ -0,0 +1,21 @@
+import { D3GraphCanvas } from '../../SPARQL/View/D3GraphCanvas';
```

Comment:
just thinking but this class is empty (like SparqlWorkspaceQueryWindow) and is just here to indicate to SparqlWorkspaceQueryWindow that its D3GraphCanvas data is structured in a certain way (Workspace). Maybe put in SparqlWorkspaceQueryWindow the fuction that know how to manipulate data of D3GraphCanvas and remove D3WorkspaceCanvas and Workspace class ? (less class more readable)